sdcm/cluster_docker.py

```diff
@@ -350,7 +350,12 @@ def __init__(self,
                                n_nodes=n_nodes,
                                params=params)
 
-    def node_setup(self, node, verbose=False, db_node_address=None, **kwargs):
+    def node_setup(self, node: DockerNode, verbose=False, db_node_address=None, **kwargs):
+        node.wait_ssh_up(verbose=verbose)
+        node.remoter.sudo("apt update", verbose=True, ignore_status=True)
```

Comment:
why do you need to do apt update here?

Reply:
Docker images usually clean the apt cache to save space, hence update is needed.